package.json

```diff
@@ -9,22 +9,23 @@
   "repository": "adobe/node-fetch-retry",
   "license": "Apache-2.0",
   "dependencies": {
+    "@adobe/fetch": "^3.3.1",
     "abort-controller": "^3.0.0",
-    "node-fetch": "^2.6.7"
+    "node-fetch": "^2.6.8"
   },
   "devDependencies": {
     "@adobe/eslint-config-asset-compute": "^1.3.3",
-    "@semantic-release/git": "^9.0.1",
+    "@semantic-release/git": "^10.0.1",
     "codecov": "^3.8.3",
     "conventional-changelog-eslint": "^3.0.9",
-    "eslint": "^6.8.0",
+    "eslint": "^8.32.0",
     "get-port": "^5.1.1",
     "license-checker": "^25.0.1",
-    "mocha": "^8.4.0",
-    "nock": "^13.2.4",
+    "mocha": "^10.2.0",
+    "nock": "^13.3.0",
     "nyc": "^15.1.0",
-    "rewire": "^5.0.0",
-    "semantic-release": "^17.4.7"
+    "rewire": "^6.0.0",
+    "semantic-release": "^20.0.2"
```

Comment:
typically we will update major versions of dev dependencies in follow-up PRs in case they require additional changes. For instance sometimes semantic-release updates require nodejs verion increases and we won't notice that until we actually try to release, (happened before, had to update to use nodejs 16: https://github.com/adobe/node-fetch-retry/blob/master/.github/workflows/node.js.yml#L47)

Reply:
ah yep, nodejs18 is required for v20+: https://github.com/semantic-release/semantic-release/releases/tag/v20.0.0